crates/aptos/src/account/create.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Aptos
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::common::types::{CliCommand, CliTypedResult, TransactionOptions};
+use crate::common::types::{CliCommand, CliTypedResult, TransactionOptions, TransactionSummary};
```

Comment:
Does the transaction summary still print the entire writeset or similar? This is pretty bad in the case of move publish.

Reply:
It doesn't print writesets anymore.  They were basically completely illegible.

Now, we only make special types if we want to extract information from the writeset / events specifically.  Otherwise, users can go to the explorer to see this information.